ScoutSuite/frontend/src/components/Breadcrumb/index.js

```diff
@@ -14,14 +14,13 @@ const Breadcrumb = () => {
   const params = useParams();
 
   return (
-    <ErrorBoundary>
```

Comment:
Don't remove the ErrorBoundary!

Reply:
it was an issue after a merge conflict, i re-added what disappeared

Reply:
Ok! I approved, you can merge when it's re-added :)